integration-tests/run-books/OCR.md

```diff
@@ -0,0 +1,252 @@
+Example:
+
+```bash
+gh workflow run "run-selected-e2e-tests.yml" \
+--ref develop \
+-f chainlink_version="v2.17.0-beta0" \
+-f test_ids="smoke/ocr2_test.go:*" \
+-f workflow_run_name="Smoke:OCR2:2.17.0-beta0" \
+-f test_secrets_override_key=YOUR_TEST_SECRETS_ID \
+-f test_config_override_path=./testconfig/ocr2/overrides/base_sepolia.toml
+```
+
+#### SMOKE
+
+##### Docker
+
+Refer [COMMON COMMANDS#Docker](#docker). Override `<./path/to/_test.go file>` as follows:
+
+With forwarders:
+`./smoke/forwarder_ocr_test.go`
+`./smoke/forwarder_ocr2_test.go`
+
+No forwarders:
+`./smoke/ocr_test.go`
+`./smoke/ocr2_test.go`
+
+##### CI
+
+Refer [COMMON COMMANDS#CI](#ci). Override `test_ids` as follows:
+
+`smoke/ocr_test.go:*` - all OCR1 tests
+`smoke/ocr2_test.go:*` - all OCR2 tests
+
+#### SOAK
+
+> [!IMPORTANT]
+> These tests require logging in to Kubernetes cluster (`aws sso login`).
+> Do not use `-timeout` flag for Soak tests with Go command. It is set in `overrides.toml`
+
+Refer [Test config README](../testconfig/README.md) for more details about Soak tests configuration.
+
+##### Kubernetes
+
+Refer [COMMON COMMANDS#Kubernetes](#kubernetes). Override path as follows:
+
+With forwarders:
+`-run 'TestForwarderOCRv1Soak' ./soak` or `make test_soak_forwarder_ocr1`
+`-run 'TestForwarderOCRv2Soak' ./soak` or `make test_soak_forwarder_ocr2`
+
+No forwarders:
+`-run 'TestOCRv1Soak' ./soak` or `make test_soak_ocr`
+`-run 'TestOCRv2Soak' ./soak` or `make test_soak_ocr2`
+
+With reorg below finality and `FinalityTagEnabled=false`:
+`-run 'TestOCRSoak_GethReorgBelowFinality_FinalityTagDisabled' ./soak` or `make test_soak_ocr_reorg_1`
+
+With reorg below finality and `FinalityTagEnabled=true`:
+`-run 'TestOCRSoak_GethReorgBelowFinality_FinalityTagEnabled' ./soak` or `make test_soak_ocr_reorg_2`
+
+With gas spike:
+`-run 'TestOCRSoak_GasSpike' ./soak` or `make test_soak_ocr_gas_spike`
+
+With change of a block gas limit (creating block congestion):
+`-run 'TestOCRSoak_ChangeBlockGasLimit' ./soak` or `make test_soak_ocr_gas_limit_change`
+
+All RPCs get down for all nodes:
+`-run 'TestOCRSoak_RPCDownForAllCLNodes' ./soak` or `make test_soak_ocr_rpc_down_all_cl_nodes`
+
+50% of nodes get RPCs down:
+`-run 'TestOCRSoak_RPCDownForHalfCLNodes' ./soak` or `make test_soak_ocr_rpc_down_half_cl_nodes`
+
+##### CI
+
+Use [On Demand OCR Soak Test](https://github.com/smartcontractkit/chainlink/actions/workflows/on-demand-ocr-soak-test.yml) workflow in GitHub.
+
+OR
+
+Run [On Demand OCR Soak Test](https://github.com/smartcontractkit/chainlink/actions/workflows/run-on-demand-ocr-soak-test.yml) workflow with `gh` as follows:
+
+```bash
+gh workflow run "on-demand-ocr-soak-test.yml" \
+--ref <branch> \
+-f chainlink_version="v<your.version>" \ # Optional, default is image created from develop branch. Not needed if you run tests against existing environment
+-f workflow_run_name="Any name" \ # Optional
+-f test_ids="<test_ID>" \ # see /chainlink/.github/e2e-tests.yml for IDS
```

Comment:
Just noticing that we mention test_ids but on example we use test to run. I do see a test id for ocr soak in e2e.yml.
Is this intentional or do we want to include both possibilities in examples?

Reply:
Thanks, @joaoluisam , fixed.